libecho/src/IAudioConverter.h

```diff
@@ -27,16 +44,28 @@ class IAudioConverter {
         return std::vector<char>(result.data(), result.data() + result.size());
     }
 
-    virtual std::vector<uint8_t> decode(std::vector<char> data) = 0;
+    /**
+     * @brief Decodes audio into primary data form.
+     */
+    virtual std::vector<uint8_t> decode(const std::vector<char> &data) = 0;
 
 protected:
-    size_t encryptedByteSize;
-    QAudioFormat const inputFormat, outputFormat;
-    double const windowSize;
+    size_t encryptedByteSize;        /**< Says how much space of type T we need to encode one byte of data. */
+    const QAudioFormat inputFormat;  /**< Audio stream output format. */
+    const QAudioFormat outputFormat; /**< Audio stream output format. */
+    const int
+        windowSize; /**< Says how much of bitrate we want to use to play sound in which unit od data is encoded. */
```

Comment:
Maybe different formatting here? windowSize in previous line, comment aligned with previous comments, break the comment into 2 lines to avoid clang-format changing it (and having too long lines :D )